src/index.php

```diff
@@ -42,7 +42,7 @@
             <div class="modal-content">
                 <div class="modal-header">
                     <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-                    <h4>Use PIN to login</h4>
+                    <h3>Use PIN to login</h3>
```

Comment:
Could you please post a screenshot of these modal-forms with the new header size? `h4` is defined by bootstrap so this could maybe lead to unwanted outcome.

Reply:
![image](https://user-images.githubusercontent.com/7527594/35471680-7636a894-0360-11e8-86a8-8936556802d6.png)

Reply:
It's a bit bigger, but ok for me.